articles/migrations/guides/management-api-v1-v2.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Migrate from Management API v1 to v2
+description: Learn how to migrate from Auth0 Management API v1 to v2
+topics:
+  - management api
+contentType: how-to 
+useCase:
+  - management api
+---
+
+# Migrate from Management API v1 to v2
+
+Auth0’s Management API v1 was deprecated 2016 and replaced with the [Auth0 Management API v2](https://auth0.com/docs/api/management/v2/). Management API v1 will reach its End Of Life on **July 6th, 2020**. Requests will begin failing with a `410` HTTP status code on or after that date.
+
+## Am I affected by the migration?
+
+Affected customers are those who meet all of the following criteria:
+* Created before {UPDATE}
+* Actively making requests to Auth0 endpoints directly under the `/api/` path (excluding those listed in https://auth0.com/docs/api/management/v1/use-cases)
+
+The following tenants are NOT affected:
+* Tenants created after {UPDATE}
+* Exclusively using the Auth0 Management API v2 endpoints 
+* The Authentication API is not affected by this deprecation
+* Only using endpoints with no API v2 alternative. See https://auth0.com/docs/api/management/v1/use-cases for more information.
```

Comment:
Not sure if we will keep enterpriseconnections there.
We already have a very few number of customers using it, they are identified and we can send separate comms and allow them to continue using. (this should be communicat